0_portfolio_input_check_functions.R

```diff
@@ -1514,4 +1511,27 @@ add_other_to_sector_classifications <- function(audit){
     mutate(sector = ifelse(sector %in% c("Industrials", "Energy", "Utilities", "Materials"), paste0("Other ", sector), sector))
   
   audit
-}
\ No newline at end of file
+}
+
+
+add_bics_sector <- function(portfolio, comp_fin_data, debt_fin_data){
+  #join in bics sectors for EQ and CB
+  portfolio_eq <- portfolio %>% filter(asset_type == "Equity") %>% 
```

Comment:
Will this still work if the filter reduces the portfolio to 0 lines?

Reply:
Yes, tested it by filtering out all equity holdings before running this. Goes through without complaints and will let you know that there's no EQ in portfolio when trying to create the portfolio subset in the next step. As before.